src/main/java/Storage.java

```diff
@@ -0,0 +1,64 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+
+/**
+ * Deals with reading from and writing to file based on inputted file path.
+ */
+public class Storage {
+    protected String filePath;
+
+    /**
+     * Creates a Storage object.
+     * @param filePath indicates where to read / write the file from / to.
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Returns an Array:ist of String pertaining to task objects in their string format.
+     * @return an Array:ist of String
+     * @throws DukeException if the file cannot be found
+     */
+    public ArrayList<String> load() throws DukeException {
```

Comment:
Is there a reason why you chose to parse the file data in your TaskList constructor instead of in this method?